imnodes.h

```diff
@@ -116,6 +116,7 @@ void Shutdown();
 // Returns the global style struct. See the struct declaration for default
 // values.
 Style& GetStyle();
+ImVec2 GeContextPanning();
```

Comment:
To be more consistent with the naming in the rest of this file, I would move this up underneath `EditorContextResetPanning` and rename this to `EditorContextGetPanning` 🙂

Reply:
Good idea!